src/App.tsx

```diff
@@ -2,14 +2,51 @@ import { BrowserRouter as Router, Routes, Route } from "react-router";
 import Home from "./pages/Home";
 import Favorites from "./pages/Favorites";
 import { PageLayout } from "./layouts/PageLayout";
+import { useState, useEffect } from "react";
 
 function App() {
+  const [isMobileView, setIsMobileView] = useState(false);
+  const [selectedCharacter, setSelectedCharacter] = useState(false);
```

Comment:
Try to add a prefix to the variables that are booleans as:

```suggestion
  const [isSelectedCharacter, setIsSelectedCharacter] = useState(false);
```

ref: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig